main.go

```diff
@@ -15,6 +15,7 @@ import (
 	"go.keploy.io/server/v2/utils"
 	"go.keploy.io/server/v2/utils/log"
 	//pprof for debugging
+	// "net/http"
```

Comment:
is this needed?

Reply:
This is needed for the sed command so that user can just use that to uncomment the import